hydra-module.nix

```diff
@@ -171,7 +171,7 @@ in
       buildMachinesFiles = mkOption {
         type = types.listOf types.path;
         default = optional (config.nix.buildMachines != []) "/etc/nix/machines";
-        example = [ "/etc/nix/machines" "/var/lib/hydra/provisioner/machines" ];
+        example = [ "/etc/nix/machines" "${baseDir}/provisioner/machines" ];
         description = "List of files containing build machines.";
```

Comment:
We probably shouldn't use a variable in the example value. However, this is overall a nice cleanup :+1:.